src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPicker.jsx

```diff
@@ -25,7 +27,8 @@ MediaPicker.propTypes = {
             type: PropTypes.string.isRequired
         })
     }).isRequired,
+    formik: PropTypes.object.isRequired,
```

Comment:
I think the imageData on top doesn't make sense anymore, since this component is just a wrapper and the query is done by the MediaPickerFilled directly now